File: app/src/main/java/com/pgssoft/mvvm/services/api/ApiInterface.java
package com.pgssoft.mvvm.services.api;

import com.pgssoft.mvvm.model.api.ApiTable;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by bstokrocki on 30.01.2017.
 */
public interface ApiInterface {
    @GET("exchangerates/tables/A?format=json")
    Call<ApiTable[]> getRates();
}

File: kotlin/app/src/main/java/com/pgssoft/kotlinplayground/service/api/ApiInterface.kt
package com.pgssoft.kotlinplayground.service.api

import com.pgssoft.kotlinplayground.model.api.ApiTable
import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by bstokrocki on 30.01.2017.
 */
interface ApiInterface {
    @GET("exchangerates/tables/A?format=json")
    fun getRates(): Call<Array<ApiTable>>
}
